Assignments/stu9/data-modeling.sql

```diff
@@ -1,7 +1,51 @@
 /**
  * Ex. 11
  */
+-- This script was generated by a beta version of the ERD tool in pgAdmin 4.
+-- Please log an issue at https://redmine.postgresql.org/projects/pgadmin4/issues/new if you find any bugs, including reproduction steps.
+BEGIN;
 
+
+CREATE TABLE IF NOT EXISTS public.lemonade
+(
+    lemonade_id serial NOT NULL,
+    lemonade_size smallint,
+    flavor_id smallint,
+    PRIMARY KEY (lemonade_id)
```

Comment:
Looks good, but I think we need to add a day (or date) field on this table to keep track of the lemonade sales per day.